### PR TITLE
gha: also test and lint stubs

### PR DESCRIPTION
**Pull request by thaJeztah**
While we were cross-compiling the code, we did not run the tests for the stubs. This patch adds a check, running on macOS to verify that the stubs work as intended.

### CODE REVIEW

**Review by rhatdan — DISMISSED** [review]
LGTM

**Comment by rhatdan** [comment]
@lsm5 @vrothberg @giuseppe @kolyshkin PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by thaJeztah** [comment]
👍 fixed; should be good now 👍

File: go-selinux/selinux_linux.go [hunk 1]
```diff
@@ -1113,6 +1115,28 @@ func chcon(fpath string, label string, recurse bool) error {
...
+	if cLabel, err := lFileLabel(fpath); err == nil && cLabel == label {
+		fastMode = true
+	}
+	return pwalkdir.Walk(fpath, func(p string, _ fs.DirEntry, _ error) error {
+		if fastMode {
+			if cLabel, err := lFileLabel(fpath); err == nil && cLabel == label {
+				return nil
+			}
+		}
+		e := lSetFileLabel(p, label)
```
[the remark this answers is not in the document]
Comment (thaJeztah) [reply go-selinux/selinux_linux.go:1131]: Ah, yes, I should do; thought I changed this one, but perhaps it got in another branch

**Review by kolyshkin — APPROVED** [review]
LGTM

**Pull request merged**